utils/release/push_to_artifactory

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import os
+import re
+import requests
+import time
+
+from artifactory import ArtifactorySaaSPath
+
+
+# Necessary ENV variables
+def getenv(name, default=None):
+    env = os.getenv(name, default)
+    if env is not None:
+        return env
+    raise KeyError("necessary {} environment is not set".format(name))
+
+
+TEMP_PATH = getenv("TEMP_PATH", ".")
+# One of the following ENVs is necessary
+JFROG_API_KEY = getenv("JFROG_API_KEY", "")
+JFROG_TOKEN = getenv("JFROG_TOKEN", "")
+
+
+class Packages(object):
+    rpm_arch = dict(all="noarch", amd64="x86_64")
+    packages = (
+        ("clickhouse-client", "all"),
+        ("clickhouse-common-static", "amd64"),
+        ("clickhouse-common-static-dbg", "amd64"),
+        ("clickhouse-server", "all"),
+        ("clickhouse-test", "all"),
+    )
+
+    def __init__(self, version: str):
+        self.deb = tuple(
+            "_".join((name, version, arch + ".deb")) for name, arch in self.packages
+        )
+
+        rev = "2"
+        self.rpm = tuple(
+            "-".join((name, version, rev + "." + self.rpm_arch[arch] + ".rpm"))
+            for name, arch in self.packages
+        )
+
+    def arch(self, deb_pkg: str) -> str:
+        if not deb_pkg in self.deb:
+            raise ValueError("{} not in {}".format(deb_pkg, self.deb))
+        return deb_pkg.removesuffix(".deb").split("_")[-1]
+
+    @staticmethod
+    def path(package):
+        return os.path.join(TEMP_PATH, package)
+
+
+class S3(object):
+    template = (
+        "https://s3.amazonaws.com/"
+        # "clickhouse-builds/"
+        "{bucket_name}/"
+        # "33333/" or "0/"
+        "{pr}/"
+        # "2bef313f75e4cacc6ea2ef2133e8849ecf0385ec/"
+        "{commit}/"
```

Comment:
I hope we will fix it soon.

Reply:
What exactly? Do we want to have another bucket template for build's artifacts?

Reply:
Just get rid of `clang-13_relwithdebuginfo_memory_bundled_unsplitted_notidy_without_coverage_deb`